### PR TITLE
[config, omni, dis] fix: quick fix for sft of Wan2.1-I2V-14B-480P

### PR DESCRIPTION
**Pull request by zbian99**
### What does this PR do?

> Add **concise** overview of what this PR aims to achieve or accomplish. Reference related GitHub issues and PRs that help with the review.

1. **Adds missing training parameters:** Introduces `vit_lr` and `train_architecture` to the training arguments definition (`veomni/utils/arguments.py`) and adds `vit_lr` to the default configuration for fine-tuning (`configs/dit/wan_sft.yaml`). This allows for differential learning rates for the Vision Transformer (ViT) encoder and specifies the training strategy (e.g., 'full' SFT or 'lora').
    
2. **Fixes a distributed communication bug:** Corrects the usage of the communication interface in the Ulysses sequence parallel implementation (`veomni/distributed/sequence_parallel/ulysses.py`) by changing `_all_gather` to `_all_gather_into_tensor` within the `_Slice.backward` function. This aligns the output tensor shape and type with existing code expectations for sequence parallelism.

### Checklist Before Starting

- [x] Search for similar PRs. Paste at least one query link here: ...
- [x] Format the PR title as `[{modules}] {type}: {description}` (This will be checked by the CI)
  - `{modules}` include `misc`, `ci`, `config`, `docs`, `core`, `data`, `dist`, `omni`, `logging`, `model`, `optim`, `ckpt`, `release`, `task`, `
  - If this PR involves multiple modules, separate them with `,` like `[megatron, torch, liger-kernals,fsdp, doc]`
  - `{type}` is in `feat`, `fix`, `refactor`, `chore`, `test`
  - If this PR breaks any API (CLI arguments, config, function signature, etc.), add `[BREAKING]` to the beginning of the title.
  - Example: `[BREAKING][fsdp, torch, liger-kernals] feat: dynamic batching`

### Test

> For changes that can not be tested by CI (e.g., algorithm implementation, new model support), validate by experiment(s) and show results like training curve plots, evaluation results, etc.

### API and Usage Example

> Demonstrate how the API changes if any, and provide usage example(s) if possible.

```python
# Add code snippet or script demonstrating how to use this
```

### Design & Code Changes

> Demonstrate the high-level design if this PR is complex, and list the specific changes.

### Checklist Before Submitting

> [!IMPORTANT]
> Please check all the following items before requesting a review, otherwise the reviewer might deprioritize this PR for review.

- [x] Read the [Contribute Guide](https://github.com/ByteDance-Seed/VeOmni/blob/main/CONTRIBUTING.md).
- [x] Apply [pre-commit checks](https://github.com/ByteDance-Seed/VeOmni/blob/main/CONTRIBUTING.md?plain=1#L22): `make commit && make style && make quality`
- [ ] Add / Update [the documentation](https://github.com/ByteDance-Seed/VeOmni/blob/main/docs).
- [ ] Add unit or end-to-end test(s) to [the CI workflow](https://github.com/ByteDance-Seed/VeOmni/tree/main/.github/workflows) to cover all the code. If not feasible, explain why: ...

### CODE REVIEW

**Comment by onehaitao** [comment]
label ascend

**Comment by onehaitao** [comment]
can submit code for wan2.1 on NPU simultaneously

**Comment by zbian99** [comment]
> can submit code for wan2.1 on NPU simultaneously

Submitted in PR #246
